### PR TITLE
a lot of new block mappings

### PR DESCRIPTION
**Pull request by eristhea**
my first medium-size pr! please be thorough with reviews, it's likely to have some errors  
  
I tried to go through the `net.minecraft.block` package and map everything I understood

### CODE REVIEW

File: mappings/net/minecraft/block/BambooBlock.mapping [hunk 1]
```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/unmapped/C_romdhvev net/minecraft/block/BambooBlock
```
Comment (OroArmor) [inline mappings/net/minecraft/block/BambooBlock.mapping:1]: Did we lose the bamboo block mapping?!?!?

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/block/BambooBlock.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/net/minecraft/block/BambooBlock.mapping:1]: yes I was confused about that
most of it directly copied

Comment (OroArmor) [reply mappings/net/minecraft/block/BambooBlock.mapping:1]: sounds good!

File: mappings/net/minecraft/block/Blocks.mapping [hunk 2]
```diff
@@ -85,6 +85,11 @@ CLASS net/minecraft/unmapped/C_qceyzrkd net/minecraft/block/Blocks
 		ARG 2 pos
 	METHOD m_suyzsusa (Lnet/minecraft/unmapped/C_txtbiemp;)I
 		ARG 0 state
+	METHOD m_sxxhsshv createLogBlock (Lnet/minecraft/unmapped/C_ifcgjcqy;Lnet/minecraft/unmapped/C_ifcgjcqy;Lnet/minecraft/unmapped/C_aevintex;[Lnet/minecraft/unmapped/C_kksdgidr;)Lnet/minecraft/unmapped/C_djwplmtn;
+		ARG 0 uprightColour
+		ARG 1 sidewaysColour
```
Comment (Leo40Git) [inline mappings/net/minecraft/block/Blocks.mapping]:
```suggestion
		ARG 0 uprightColor
		ARG 1 sidewaysColor
```
Use US English spellings for consistency, please!

File: mappings/net/minecraft/block/Blocks.mapping [hunk 3]
```diff
@@ -126,6 +131,9 @@ CLASS net/minecraft/unmapped/C_qceyzrkd net/minecraft/block/Blocks
 		ARG 1 block
 	METHOD m_yjvwdaey (ILnet/minecraft/unmapped/C_txtbiemp;)I
 		ARG 1 state
+	METHOD m_yswbpyos createLogBlock (Lnet/minecraft/unmapped/C_ifcgjcqy;Lnet/minecraft/unmapped/C_ifcgjcqy;)Lnet/minecraft/unmapped/C_djwplmtn;
+		ARG 0 uprightColour
+		ARG 1 sidewaysColour
```
Comment (Leo40Git) [inline mappings/net/minecraft/block/Blocks.mapping]:
```suggestion
		ARG 0 uprightColor
		ARG 1 sidewaysColor
```

**Review by Leo40Git — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/block/Blocks.mapping (hunk 2 above)
Comment (eristhea) [reply mappings/net/minecraft/block/Blocks.mapping]: done!

**Review by Leo40Git — APPROVED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/block/enums/Instrument.mapping [hunk 4]
```diff
@@ -1,7 +1,15 @@
 CLASS net/minecraft/unmapped/C_onjtjtsi net/minecraft/block/enums/Instrument
 	FIELD f_jpklnqjf sound Lnet/minecraft/unmapped/C_avavozay;
+	FIELD f_sjascaer isMobHead Z
```
Comment (IotaBread) [inline mappings/net/minecraft/block/enums/Instrument.mapping]:
```suggestion
	FIELD f_sjascaer mobHead Z
```

File: mappings/net/minecraft/block/enums/Instrument.mapping [hunk 5]
```diff
@@ -1,7 +1,15 @@
...
 	FIELD f_wxktsuqr name Ljava/lang/String;
 	METHOD <init> (Ljava/lang/String;ILjava/lang/String;Lnet/minecraft/unmapped/C_avavozay;)V
 		ARG 3 name
 		ARG 4 sound
+	METHOD <init> (Ljava/lang/String;ILjava/lang/String;Lnet/minecraft/unmapped/C_avavozay;Z)V
+		ARG 3 name
+		ARG 4 sound
+		ARG 5 isMobHead
```
Comment (IotaBread) [inline mappings/net/minecraft/block/enums/Instrument.mapping]:
```suggestion
		ARG 5 mobHead
```

**Review by IotaBread — APPROVED** [review]

File: mappings/net/minecraft/block/enums/Instrument.mapping (hunk 4 above)
Comment (eristhea) [reply mappings/net/minecraft/block/enums/Instrument.mapping]: looks like I need to reread the conventions

**Review by supersaiyansubtlety — APPROVED** [review]

**Pull request merged**